### ISSUE TITLE
Lookup major junos version number and make available for diffing

### ISSUE THREAD

**Issue by jonstout** [GlobalNOC/OESS#605]
in `get_lsp_paths` (which is required for detecting circuit's path) the current xml path isn't valid from junos version 17 on.

The change required is from
```
my $next_hops = $xp->find('./r:explicit-route/r:address', $path);
```

To
```
my $next_hops = $xp->find('./r:explicit-route/r:explicit-route-element/r:address', $path);
```

To handle this change we should check the major junos version number which should be the first number before the first period. For example `17` would be  found from version `17.3R3.9`.

**Comment by apertome** [comment]
This should be fixed; PR opened

**Comment by ajragusa** [comment]
fixed in pull request #616
